pkg/cmd/cli/cmd/generate.go

```diff
@@ -109,3 +95,22 @@ $ s2i generate docker.io/centos/nodejs-10-centos7 Dockerfile.gen
 
 	return generateCmd
 }
+
+func manageConfigImageLabelsBuildImageName(cfg *api.Config) error {
```

Comment:
Create the context object when the root command is invoked, and pass it through here:

```suggestion
func manageConfigImageLabelsBuildImageName(ctx context.Context, cfg *api.Config) error {
```